web-app/server/routes/me.js

```diff
@@ -0,0 +1,256 @@
+          let subjects = responseStudent.msg.Subjects;
+          let allSubject = responseAllSubject.msg;
+
+          // Duyet qua tat ca Subject de tim lay ra object subject ma student theo hoc
+          for (let i = 0; i < subjects.length; i++) {
+            for (let k = 0; k < allSubject.length; k++) {
+              if (subjects[i] === allSubject[k].SubjectID) {
+                subjectOfStudent.push(allSubject[k]);
+              }
+            }
+          }
+
+          res.json({
+            success: true,
+            subjects: subjectOfStudent
+          });
+        } else {
+          res.json({
+            success: false,
+            msg: response.msg
+          });
+        }
+      }
+    });
+  } else if (user.role === USER_ROLES.TEACHER) {
+    User.findOne({ username: user.username, role: USER_ROLES.TEACHER }, async (err, teacher) => {
+      if (err) {
+        res.json({
+          success: false,
+          msg: err
+        });
+      } else {
+        const networkObj = await network.connectToNetwork(user);
+
+        if (!networkObj) {
+          res.json({
+            success: false,
+            msg: 'Failed connect to blockchain',
+            status: 500
+          });
+        }
+
+        const responseTeacher = await network.query(networkObj, 'QueryTeacher', user.username);
+        const responseAllSubject = await network.query(networkObj, 'GetAllSubjects');
+        let subjectOfTeacher = [];
+
+        if (response) {
+          let subjects = responseTeacher.msg.Subjects;
+          let allSubject = responseAllSubject.msg;
+
+          // Duyet qua tat ca Subject de tim lay ra object subject ma teacher co
+          for (let i = 0; i < subjects.length; i++) {
+            for (let k = 0; k < allSubject.length; k++) {
+              if (subjects[i] === allSubject[k].SubjectID) {
+                subjectOfTeacher.push(allSubject[k]);
+              }
+            }
+          }
+
+          res.json({
+            success: true,
+            subjects: subjectOfTeacher
+          });
+        } else {
+          res.json({
+            success: false,
+            msg: response.msg
+          });
+        }
+      }
+    });
+  } else {
+    res.json({
+      success: true,
+      msg: 'You do not have subject'
+    });
+  }
+});
+
+router.get('/certificate', async (req, res) => {
+  const user = req.decoded.user;
+
+  if (user.role === USER_ROLES.STUDENT) {
+    User.findOne({ username: user.username, role: USER_ROLES.STUDENT }, async (err, student) => {
+      if (err) {
+        res.json({
+          success: false,
+          msg: err
+        });
+      } else {
+        const networkObj = await network.connectToNetwork(user);
+
+        if (!networkObj) {
+          res.json({
+            success: false,
+            msg: 'Failed connect to blockchain',
+            status: 500
+          });
+        }
+
+        const response = await network.query(networkObj, 'GetAllCertificates');
```

Comment:
@tantv-918 sửa chaincode function name
```suggestion
getMyCerts
```
@conglt10 sửa logic